Add a news post for `osu!CenterStage - Idol 2021` (#5775)
ppy/osu-wiki: news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open.md

---
layout: post
title: "osu!CenterStage - Idol 2021"
date: 2021-07-02 12:00:00 +0000
---

> We are happy to announce the yearly idol contest is back and ready for action!
> ![](/wiki/shared/news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open/centerstagebanner.jpg)
> A new chapter in the history of this competition, but the spirit is as strong as ever!

Reviewer: ```suggestion
We are happy to announce the yearly idol contest is back and ready for action!

![](/wiki/shared/news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open/centerstagebanner.jpg)

A new chapter in the history of this competition, but the spirit is as strong as ever!
```

> ### **What is osu!CenterStage?**

Reviewer: ```suggestion
## What is osu!CenterStage?

```

A successor to the much beloved osu!idol competition which concluded in 2020. The format remains largely the same. Participants will sing through the various rounds set for them until a winner is decided. A panel of judges will guide the direction of this contest by evaluating and critiquing the participant's performances.

> ### **How does this work?**

Reviewer: ```suggestion
## How does this work?

```

For the auditions, record yourself performing a 30-60 second sample of any song (with no BGM) then submit in your entry. It's as simple as that! July 21st will be the deadline so make haste and submit in those wonderful entries!

> ### **Any Prizes?**

Reviewer: ```suggestion
## Any prizes?

```

Why yes we do. We have some rewards in store for those who come out on top.

- **Winner** - osu!supporter tag for 8 months
- **Runner-Up** - osu!supporter tag for 6 months
- **Third Place** - osu!supporter tag for 4 months

> For all relevant information you will need to know in entering this year's competition check out the [*official thread*](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receiving important updates regarding this contest, join the official [*discord server*](https://discord.gg/eBaDTYrvby).

Reviewer: ```suggestion
For all relevant information you will need to know in entering this year's competition check out the [official thread](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receiving important updates regarding this contest, join the official [Discord server](https://discord.gg/eBaDTYrvby).
```

---
layout: post
title: "osu!CenterStage - Idol 2021"
date: 2021-07-02 12:00:00 +0000
---

> We are happy to announce the yearly idol contest is back and ready for action!

Reviewer: ```suggestion
We are happy to announce that the yearly idol contest is back and ready for action!
```

![](/wiki/shared/news/2021-07-02-osu!centerstage-idol-2021-auditions-now-open/centerstagebanner.jpg)

A new chapter in the history of this competition, but the spirit is as strong as ever!

## What is osu!CenterStage?

> A successor to the much beloved osu!idol competition which concluded in 2020. The format remains largely the same. Participants will sing through the various rounds set for them until a winner is decided. A panel of judges will guide the direction of this contest by evaluating and critiquing the participant's performances.

Reviewer: ```suggestion
A successor to the much beloved osu!idol competition which concluded in 2020. The format remains largely the same. Participants will sing through the various rounds set for them until a winner is decided. A panel of judges will guide the direction of this contest by evaluating and critiquing the participants' performances.
```

## How does this work?

> For the auditions, record yourself performing a 30-60 second sample of any song (with no BGM) then submit in your entry. It's as simple as that! July 21st will be the deadline so make haste and submit in those wonderful entries!

Reviewer: ```suggestion
For the auditions, record yourself performing a 30-60 second sample of any song (with no background music), then submit in your entry. It's as simple as that! July 21st will be the deadline, so make haste and submit in those wonderful entries!
```

## Any prizes?

Why yes we do. We have some rewards in store for those who come out on top.

- **Winner** - osu!supporter tag for 8 months
- **Runner-Up** - osu!supporter tag for 6 months
- **Third Place** - osu!supporter tag for 4 months

> For all relevant information you will need to know in entering this year's competition check out the [official thread](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receiving important updates regarding this contest, join the official [Discord server](https://discord.gg/eBaDTYrvby).

Reviewer: ```suggestion
For all relevant information you will need to know before entering this year's competition, check out the [official thread](https://osu.ppy.sh/community/forums/topics/1359674). To get involved with the community and receive important updates regarding this contest, join the official [Discord server](https://discord.gg/eBaDTYrvby "Discord").
```
